lib/utils/index.js

```diff
@@ -1,11 +1,16 @@
 const crypto = require('crypto');
 
-// todo: generate proper licence numbers
-const generateLicenceNumber = () => {
+const generateLicenceNumber = (type, isNew) => {
   const buf = crypto.randomBytes(48).toString('hex');
-  const chars = buf.replace(/[0-9]/g, '').substring(0, 2).toUpperCase();
-  const digits = buf.replace(/[a-z]/g, '').substring(0, 6);
-  return `${chars}-${digits}`;
+  const digits = buf.replace(/[a-z]/g, '').substring(0, 8);
```

Comment:
This is gonna generate random strings of digits, but there's no guarantee of uniqueness, so it's going to fail to update the record when you attempt to save it if there's a collision. Even with 8 digits that get's pretty likely pretty quickly due to the https://en.wikipedia.org/wiki/Birthday_problem

Reply:
@wheelsandcogs why did not we use uuid ?

Reply:
Because licence numbers are supposed to be reasonably human-comprehensible (read out over the phone etc), and so UUIDs are _really_ bad.